docs/external-node.md

```diff
@@ -0,0 +1,532 @@
+# ExternalNode
+
+## Table of Contents
+
+<!-- toc -->
+- [What is ExternalNode?](#what-is-externalnode)
+- [Prerequisites](#prerequisites)
+- [The ExternalNode resource](#the-externalnode-resource)
+  - [Name and Namespace](#name-and-namespace)
+  - [Interfaces](#interfaces)
+- [Apply Antrea NetworkPolicy to ExternalNode](#apply-antrea-networkpolicy-to-externalnode)
+  - [Antrea NetworkPolicy configuration](#antrea-networkpolicy-configuration)
+  - [Bypass Antrea NetworkPolicy](#bypass-antrea-networkpolicy)
+- [RBAC for antrea-agent](#rbac-for-antrea-agent)
+- [VM network configuration](#vm-network-configuration)
+- [OpenFlow pipeline](#openflow-pipeline)
+    - [Non-IP packet](#non-ip-packet)
+    - [IP packet](#ip-packet)
+- [Install Antrea Agent on VM](#install-antrea-agent-on-vm)
+  - [Prerequisites on Kubernetes cluster](#prerequisites-on-kubernetes-cluster)
+  - [Installation on Linux VM](#installation-on-linux-vm)
+    - [Prerequisites](#prerequisites-1)
+    - [Installation](#installation)
+  - [Installation on Windows VM](#installation-on-windows-vm)
+    - [Prerequisites](#prerequisites-2)
+    - [Installation](#installation-1)
+- [Limitations](#limitations)
+<!-- /toc -->
+
+# What is ExternalNode?
```

Comment:
IMO the heading levels are wrong (off by one level)
since you use `# ExternalNode` as the title for the document, all section titles should use a lower level (e.g., `## What is ExternalNode?`)

Reply:
Adjust in the document.